atlas/commands/bootstrap/bootstrap.go

```diff
@@ -181,6 +181,9 @@ func generateProtobuf() error {
 // initGoMod calls "go mod init" to generate .toml files
 func initGoMod() error {
 	fmt.Print("Starting mod project... ")
+	if err := runCommand("go", "mod", "tidy"); err != nil {
+		return err
+	}
 	if err := runCommand("go", "mod", "vendor"); err != nil {
 		return err
 	}
```

Comment:
There's no need for `go mod download` after `go mod vendor`.

Reply:
removed